blog/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+
+from .views import PostListView, PostDetailView, add_comment
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)